ee/clickhouse/queries/test/test_cohort_query.py

```diff
@@ -1057,7 +1058,11 @@ def test_cohort_filter(self):
         p1 = Person.objects.create(
             team_id=self.team.pk, distinct_ids=["p1"], properties={"name": "test", "name": "test"}
         )
-        cohort = _create_cohort(team=self.team, name="cohort1", groups=[{"properties": {"name": "test"}}])
```

Comment:
How will the current system handle instances of this group pattern now?

Reply:
Doesn't exist, as far as I can tell. Only in our tests that we borked things up

Reply:
hmmmm ok